ansible/group_vars/all/defaults.yaml

```diff
@@ -5,7 +5,10 @@ python_path: ""
 # If it's not there, the kubernetes_full_version will have "None" for a version number.
 #
 # IMPORTANT When you update kubernetes_version, also update crictl_version.
-kubernetes_version: "1.28.5"
+kubernetes_major: "1"
+kubernetes_minor: "28"
+kubernetes_patch: "6"
+kubernetes_version: "1.28.6"
```

Comment:
It would be good to keep only one variable `kubernetes_version` . Ideally we want to remove this variable and only keep it defaulted in CLI in future iterations.

Reply:
done